xrviz/tests/test_control.py

```diff
@@ -10,4 +10,4 @@ def control(data):
 
 def test_control_initial(control):
     tabs = [tab.name for tab in control.tabs]
-    assert tabs == ['Fields', 'Set Coords']
+    assert tabs == ['Fields', 'Projection', 'Set Coords']
```

Comment:
This is only the case under your system and Travis, when we've controlled the environment. For other people it would fail, if they don't have cartopy.

Reply:
I has been handled now.